lua/hover/providers/jira.lua

```diff
@@ -0,0 +1,39 @@
+local job = require('hover.async.job').job
+
+local function enabled()
+    -- Match 2 or more uppercase letters followed by a '-' and 1 or more digits.
+    return vim.fn.expand('<cWORD>'):match('%u%u+-%d+') ~= nil
+end
+
+local function execute(done)
+    local bufnr = vim.api.nvim_get_current_buf()
+    local cwd = vim.fn.fnamemodify(vim.api.nvim_buf_get_name(bufnr), ':p:h')
```

Comment:
Is this necessary?

Reply:
I don't know tbh, I started with a copy of the GitHub provider and it has this for getting the cwd.
I know there is a function (vim.fn.getcwd I believe) for it but I figured the way the GitHub provider does it would have a reason.

Reply:
The github provider needs to set the cwd since the `gh` command looks into the `.git` folder. I don't think we need to do that for jira.

Reply:
Ah oke, no that is not needed for Jira, I will try simplify this.

Reply:
The cwd part has been removed.